converter/biosamples.py

```diff
@@ -0,0 +1,67 @@
+from biosamples_v4.models import Sample, Attribute
+from json_converter.json_mapper import JsonMapper
+
+
+def get_concrete_type(schema_url):
+    concrete_type = schema_url.split('/')[-1]
+    return concrete_type
+
+
+ATTRIBUTE_SPEC = {
+    'Biomaterial Core - Biomaterial Id': ['content.biomaterial_core.biomaterial_id'],
+    'HCA Biomaterial Type': ['content.describedBy', get_concrete_type],
```

Comment:
Is the `get_concrete_type` function a mapper for the value in 'content.describedBy'?

Reply:
I think so, I'm not sure  I get you're meaning so I'll describe what it does and hope I answer your question.

1. The `get_concrete_type` function is a post process, you can see other examples of these [here](https://github.com/ebi-ait/ingest-archiver/blob/master/archiver/dsp_post_process.py).
2. The post process is applied to value **after** the key `content.describedBy` is mapped to the value (using [this](https://api.ingest.archive.data.humancellatlas.org/biomaterials/60875444dd9aab1232d3cde6) example: `https://schema.humancellatlas.org/type/biomaterial/15.5.0/donor_organism`)
3. The value is sent to the `get_concrete_type` function which in our example returns: `donor_organism `
4. So the resulting dictionary for our example will be:
```
{
    'Biomaterial Core - Biomaterial Id': 'H9_donor',
    'HCA Biomaterial Type': 'donor_organism',
    'HCA Biomaterial UUID': '9c94b6bb-7fb0-451d-8dec-5cad455c4d24',
    'Is Living': 'yes',
    'Sex': 'female'
}
```
Does that answer your question?